ehrapy/preprocessing/_normalization.py

```diff
@@ -69,6 +75,23 @@ def _scale_func_group(
         return None
 
 
+@singledispatch
```

Comment:
What do you think about systematically introducing singledispatch in this manner? It might be a bit of an overkill here; but I think in the long run, using this structure reduces code complexity by introducing a regularly appearing pattern.

Reply:
The StandardScaler I think works also with sparse matrices. E.g. the MaxAbsScaler does not I think. For focusing on the repeating patterns in a first step, have been restrictive here in disallowing everything but `np.array` and `dask.array.Array`

Reply:
Yes, I agree with your argument